src/xcode/ENA/ENA/Source/Services/PPAccessControl/Model/PPACError.swift

```diff
@@ -20,6 +22,11 @@ enum PPACError: Error {
 			return "timeIncorrect"
 		case .timeUnverified:
 			return "timeUnverified"
+		case .minimumTimeSinceOnboarding:
+			return "minimumTimeSinceOnboarding"
```

Comment:
`"TIME_SINCE_ONBOARDING_UNVERIFIED"` as raw value?

from https://github.com/corona-warn-app/cwa-app-tech-spec/blob/proposal/self-report/docs/spec/data-donation-client-srs.md#submitting-a-self-report

Reply:
What about the previous values like timeUnverified?